### DISCUSSION TITLE
Please remove the achivements section

### DISCUSSION THREAD

**Discussion by eyalroz** [community/community#19411] [Other Feature Feedback, Questions, & Ideas] [answered]
It's not useful, it's distracting, it's silly.

**Comment by melvincwng** [comment, marked as answer]
@eyalroz, you can hide your achievements in the 'Settings' page.

Hope this clarifies.

**Reply by eyalroz** [reply]
@melvincwng : That improves things somewhat, but now it's as though some people have achievements and others don't, because they removed them? Or am I misunderstanding the idea?

**Reply by krissemicolon** [reply]
No, if you hide them, they aren't on your profile anymore. For you and everyone else.

**Comment by myConsciousness** [comment]
I believe such a badge is a good indicator of a user's level of activity on GitHub. However, it's not a good idea to flood the site with low-quality output in order to earn a badge at the same time.

**Reply by melvincwng** [reply]
@myConsciousness, well said Kato Shinya!

**Comment by rodukov** [comment]
![image](https://user-images.githubusercontent.com/101352977/175785844-a6c46695-975a-4ca5-9053-ed83c88b94c0.png)
As for me it's cool. **You can hide them from everyone in the settings**
_If I helped you, make the answer the main one. I will be pleased._

**Comment by matigodoy** [comment]
I think it is a good way to measure how much a person is pasionate about programming and contributting to the comunity
